content/blog/2023/2023-q1-update/index.md

```diff
@@ -0,0 +1,135 @@
+
+We added several new community partnerships to our managed hub service.
+We've deployed new hub infrastructure for each of the following groups:
+
+- Our [University of Toronto Jupyter service](https://act.utoronto.ca/jupyterhub-support/) now has a dedicated R hub.
+- We ran an event hub for the [Drakkar Ocean project](https://www.drakkar-ocean.eu/)
+- We deployed a hub for the [NASA Visuaslization Exploration and Data Analysis (VEDA) project](https://www.earthdata.nasa.gov/esds/veda).
+- We deployed a hub for the [QuantifiedCarbon](https://www.linkedin.com/company/quantifiedcarbon/about/) organization.
+- We ran an event hub for [OceanHackWeek 2023](https://oceanhackweek.org/).
+- We onboarded several new community colleges to our "JupyterHubs Education in Community Colleges" collaboration with [CloudBank](https://www.cloudbank.org/welcome-cloudbank) and [UC Berkeley CDSS](https://data.berkeley.edu/).
+
+## Service improvements
+
+Below are a few highlights for ways in which we improved our [Managed Cloud Service](https://docs.2i2c.org) for our partner communities.
+
+### We simplified our authentication workflow with CILogon
+
+Authentication services allow us to identify a user when they log onto a hub, which determines their ability to access hub resources.
+Previously we had used a combination of [Auth0](https://auth0.com/) and [CILogon](https://www.cilogon.org/) for authentication.
+
+However, over the past year we've been happy with our use of CILogon so far, especially because of its non-profit status and alignment with many research and education institutions that we work with.
+This quarter, we decided to streamline our authentication process by dropping the use of Auth0 and grow our partnership with CILogon.
+
+💡 _Learn more_
+
+- See [CILogon's write up about it's partnership with 2i2c](https://www.ncsa.illinois.edu/security-made-simple-with-ncsas-cilogon/)
+- See [our blog post about our use of CILogon](https://2i2c.org/blog/2023/cilogon-integration/)
+
+### We exposed user activity dashboards in JupyterHub so communities know how many people are using the service
+
+We tend to work with _leaders_ of communities that utilize our service and infrastructure for many others in their network.
+For example, a teacher with a classroom of students, or a researcher with a global network of collaborators.
+
+In these cases, it's useful to track how many users are actively using a platform over various metrics of time.
+This can tell you whether your community finds a service useful, and whether this is growing or shrinking.
+
+We use [Grafana](https://grafana.com/) to automatically generate dashboards of activity for all of our community hubs and clusters.
+However, tracking **daily, weekly, and monthly active users** was not part of JupyterHub's core functionality.
+
+So, we decided to upstream this functionality into JupyterHub and expose it via [the JupyterHub Grafana project](https://github.com/jupyterhub/grafana-dashboards).
```

Comment:
"we decided to upstream this functionality", maybe we may want to add a link to Yuvi's blog post in the Jupyter blog: https://blog.jupyter.org/accurately-counting-daily-weekly-monthly-active-users-on-jupyterhub-6fbec6c6ce2f

Reply:
jaja... forget about it, I just saw the link below...